tests/settings/test_yaml.py

```diff
@@ -37,11 +38,10 @@ def test_uuid_resolver(self):
         yaml.add_representer(uuid.UUID, uuid_representer, Dumper=TestDumper)
         uuid_add_implicit_resolver(loader=TestLoader, dumper=TestDumper)
 
-        self.assertEqual(
-            yaml.dump(uuid_in, Dumper=TestDumper),
-            'test: cc3702ca-699a-4aa6-8226-4c938f294d9b\n')
-        self.assertEqual(
-            yaml.load(UUID_OUT, Loader=TestLoader), EXPECTED_UUID_OBJ)
+        self.assertEqual(yaml.dump(uuid_in, Dumper=TestDumper),
+                         'test: cc3702ca-699a-4aa6-8226-4c938f294d9b\n')
+        self.assertEqual(yaml.load(UUID_OUT, Loader=TestLoader),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of unsafe yaml load. Allows instantiation of arbitrary objects. Consider yaml.safe_load().](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)